core/src/test/java/org/opensearch/sql/expression/json/JsonFunctionsTest.java

```diff
@@ -61,4 +76,206 @@ private ExprValue execute(ExprValue jsonString) {
     FunctionExpression exp = DSL.jsonValid(DSL.literal(jsonString));
     return exp.valueOf();
   }
+
+  @Test
+  void json_returnsJsonObject() {
+    FunctionExpression exp;
+
+    // Setup
+    final String objectJson =
+        "{\"foo\": \"foo\", \"fuzz\": true, \"bar\": 1234, \"bar2\": 12.34, \"baz\": null, "
+            + "\"obj\": {\"internal\": \"value\"}, \"arr\": [\"string\", true, null]}";
```

Comment:
Nit.

Since we already have a multi-line string, maybe we could format it so it's a bit easier to read (all the escapes already make it tough enough!) Also, what about using field names like `"string_field"` to make it a bit more self-documenting?

🤷 

```suggestion
    final String objectJson =
        "{\"string_field\": \"foo\","
          + "\"bool_field\": true,"
          + "\"int_field\": 1234,"
          + "\"float_field\": 12.34,"
          + "\"null_field\": null, "
          + "\"obj_field\": {\"internal\": \"value\"}, \"arr\": [\"string\", true, null]}";
```

Reply:
I guess this is just forked from the other branch. Oh well. Feel free to ignore.